src/datachain/catalog/catalog.py

```diff
@@ -1765,6 +1766,7 @@ def query(
             {
                 "DATACHAIN_QUERY_PARAMS": json.dumps(params or {}),
                 "DATACHAIN_JOB_ID": job_id or "",
+                "DATACHAIN_CHECKPOINTS_RESET": str(reset),
```

Comment:
Will it be `None` string if `reset` param is not set?
Let's add it to the env variables only if it is `True`?
Or, at least, `str(bool(reset))` or something like `"True" if reset else "False"`

Reply:
Yes, I agree, but same time having `None` as value in env variable kind of smells 👀😅